unified-installer/astra-unified-installer.sh

```diff
@@ -99,15 +99,38 @@ readonly __FATAL=50
 #     captured_stdout="$(curl -sS https://bad-url.com 2> "$_ERR_FILE")"
 #     captured_stderr="$(get_captured_err)"
 readonly __ERR_FILE="tmp_last_captured_error.txt"
+
+# __TMP_ENV is used to store the user's env vars so that we can then re-apply them after having sourced
+# their config file. This allows us to make command line vars take precedence over what's in the config file.
+readonly __TMP_ENV="tmp.env"
```

Comment:
Can we use a randomly generated name or something more unique like `install_script_env_file_helper.env`? I could see a tester or customer already having used “tmp.env” and us possibly overwriting it.

Reply:
Good point, I'll add some generated value to the name.